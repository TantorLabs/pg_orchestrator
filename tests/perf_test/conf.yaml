kind: perf
db_params:
  - db_type: pgdg
    db_version: 16.4
    db_port: 5432
db_initial_script: fill_db.sh
docker:
  image: ubuntu:22.04
performance_coefficient: 0.8
cases:
  - name: case_1
    explain_queries:
      - query: explain_query.sql
        expected:
          - explain_expected_1.txt
    timing_queries:
      - query: timing_query.sql
        expected_time_ms: 1500