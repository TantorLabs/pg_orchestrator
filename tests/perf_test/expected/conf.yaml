test_results:
- test_version: '16.4'
  test_edition: null
  cases:
  - case_name: case_1
    explain_results:
    - query: explain_query.sql
      result: match
      matched_expected_file: explain_expected_1.txt
    timing_results:
    - query: timing_query.sql
      result: within expected
      status: success